dhash/database.py

```diff
@@ -106,22 +106,32 @@ class HashChannel(database.base):
     __tablename__ = "fun_dhash_channels"
 
     idx = Column(Integer, primary_key=True, autoincrement=True)
-    guild_id = Column(BigInteger)
-    channel_id = Column(BigInteger)
+    guild_id = Column(BigInteger, nullable=False)
+    channel_id = Column(BigInteger, nullable=False)
+    reaction_limit = Column(Integer, nullable=False)
 
     __table_args__ = (UniqueConstraint(guild_id, channel_id),)
 
-    def add(guild_id: int, channel_id: int) -> HashChannel:
+    @staticmethod
+    def add(guild_id: int, channel_id: int, reaction_limit: int = 5) -> HashChannel:
         existing = HashChannel.get(guild_id, channel_id)
 
         if existing:
             return existing
 
-        channel = HashChannel(guild_id=guild_id, channel_id=channel_id)
+        channel = HashChannel(
+            guild_id=guild_id, channel_id=channel_id, reaction_limit=reaction_limit
+        )
         session.add(channel)
         session.commit()
         return channel
 
+    def change_reaction_limit(self, reaction_limit: int):
```

Comment:
Good, I'd just change name to 'set_limit()' to stay consistend with the rest of the tables.